src/main/java/examples/HTTPExamples.java

```diff
@@ -918,6 +918,14 @@ public void example60(Vertx vertx) {
 
   }
 
+  public void example61(Vertx vertx) {
+
+    HttpServerOptions options = new HttpServerOptions()
+      .setUseProxyProtocol(true);
+
+    HttpServer server = vertx.createHttpServer(options);
```

Comment:
I think we should add:

```java
server.requestHandler(request -> {
  // Print the actual client address provided by the HA proxy protocol instead of the proxy address
  System.out.println(request.remoteAddress());
});
```